committee-code-of-conduct/charter.md

```diff
@@ -0,0 +1,150 @@
+# Kubernetes Code of Conduct Committee Charter
+
+## Mission/Purpose
+Our primary concern is creating and maintaining a safe and respectful community.
+Our role is to provide and enforce a well-considered viewpoint on what
+constitutes acceptable behavior within our community. The [Code of
+Conduct](https://git.k8s.io/community/code-of-conduct.md) serves as the primary
+policy document and is supported with additional references and tools as needed.
+Since maintaining a safe environment is a very large part of what the committee
+does, we must carefully balance transparency of process with preserving the
+privacy of those who come to the committee or other community leaders with an
+incident report.
+
+The committee understands how challenging these matters are for everyone
+involved, and that the process is never perfect by nature of its need to serve
+everyone in the community equally. That said, the committee is oriented toward
+the protection of our community and takes that duty seriously.
+
+## Communication with the committee
+The committee maintains a private mailing list for reporting incidents, asking
+confidential questions, and internal committee communication:
+
+[conduct@kubernetes.io][email]
+
+This email alias may not be the fastest path to a response in urgent situations.
+**If a response is time critical, reaching individual committee members via
+Slack is advisable. But an email must also be sent to
+[conduct@kubernetes.io][email] for tracking purposes**. This feedback helps
+guide the implementation of new policies and procedures.
+
+### Others acting on behalf of the committee
+Community moderation administrators in Slack, the mailing list, community
+events, and elsewhere are extensions of the committee and are able to be first
+responders to incidents. They are explicitly empowered to take those actions
+necessary to protect the community, especially when no committee members are
+available. All such events must be retroactively reviewed by the committee for
+appropriateness and consistency. 
+
+Steering committee members may also act in limited cases to enforce the code of
+conduct. Examples of this may be the deletion of GitHub comments, offensive
+Slack messages, or the eviction of bad actors from public meetings. These
```

Comment:
One small point here - the steering committee might not have powers to delete GitHub comments in some repos (github admin team can do that) or delete slack messages (slack admin team can do that).

Reply:
Most do, but we should double check that it exists across all k8s-orgs.  You may want to file an issue https://github.com/kubernetes/org

Reply:
I do not agree that steering committee members should implicitly have Owner or even Admin rights across all of our orgs and all of our repos.  Should steering have the ability to ask the appropriate teams to take action? Sure.  But at least for all k8s orgs, we delegated the responsibilities to the GitHub Admin team, we should not subvert that.  We addressed this in https://github.com/kubernetes/org/issues/637